whelk-core/src/main/resources/ext/marcframe.json

```diff
@@ -17372,38 +17561,108 @@
             {"d": "1989"},
             {"w": "1217888"}
           ]}},
+          "result": {"mainEntity": {
+            "otherPhysicalFormat": [{
+              "@type": "Instance",
+              "instanceOf": {
+                "@type": "Work",
+                "marc:displayText": ["Originalversion:"],
+                "contribution": [{
+                  "@type": "PrimaryContribution",
+                  "agent": {
+                    "@type": "Agent",
+                    "label": "Andersson, Lennart"
+                  }
+                }]
+              },
+              "describedBy": [{
+                "@type": "Record",
+                "controlNumber": "1217888"
+              }],
+              "hasTitle": [{
+                "@type": "Title",
+                "mainTitle": "Umebygdens soldater 1695-1895"
+              }],
+              "provisionActivityStatement": "1989"
+            }]
+          }}
+        },
+        {
+          "name": "Example from m5z1zbbz50clqtz",
+          "source": {"776": {"ind1": "0", "ind2": "8", "subfields": [
+            {"i": "Online version:"},
+            {"a": "Adam, von Bremen, 11th cent."},
+            {"s": "Gesta Hammaburgensis ecclesiae pontificum. French."},
+            {"t": "Histoire des archevêques de Hambourg."},
+            {"d": "[Paris] : Gallimard, c1998"},
+            {"w": "(OCoLC)607718109"}
+          ]}},
           "result": {"mainEntity": {
             "otherPhysicalFormat" : [ {
               "@type" : "Instance",
-              "contribution" : [ {
-                "@type" : "PrimaryContribution",
-                "agent" : [ {
-                  "@type" : "Agent",
-                  "label" : "Andersson, Lennart"
-                } ]
-              } ],
-              "describedBy" : [ {
-                "@type" : "Record",
-                "controlNumber" : "1217888"
-              } ],
-              "hasTitle" : [ {
-                "@type" : "Title",
-                "mainTitle" : "Umebygdens soldater 1695-1895"
-              } ],
-              "marc:displayText" : [ "Originalversion:" ],
-              "marc:toDisplayNote" : true,
-              "provisionActivityStatement" : "1989"
-            } ]
+              "instanceOf": {
+                "@type": "Work",
+                "contribution" : [ {
+                  "@type": "PrimaryContribution",
+                  "agent": {
+                    "@type": "Agent",
+                    "label": "Adam, von Bremen, 11th cent."
+                  }
+                }],
+                "hasTitle": [{
+                  "@type": "Title",
+                  "mainTitle": "Gesta Hammaburgensis ecclesiae pontificum. French."
+                }],
+                "marc:displayText": ["Online version:"]
```

Comment:
I see that we wrote that this should be put on the work in issue 2769, but why? AFAIK, marc:displayText and describedBy "go together"?